pkg/helm/status.go

```diff
@@ -4,19 +4,19 @@ import (
 	"fmt"
 	"strings"
 
-	"gopkg.in/yaml.v2"
-
 	"github.com/deislabs/porter/pkg/printer"
+	yaml "gopkg.in/yaml.v2"
 )
 
 // StatusStep represents the structure of an Status action
 type StatusStep struct {
-	Description string          `yaml:"description"`
-	Arguments   StatusArguments `yaml:"helm"`
+	StatusArguments `yaml:"helm"`
 }
 
 // StatusArguments are the arguments available for the Status action
 type StatusArguments struct {
+	Step `yaml:,inline`
```

Comment:
I purposefully included outputs here (it wasn't before). Could be neat to let someone do a status and get a piece of data out, and plumb that into another step.